src/main/proto/example/hello_world/v1/hello_world_service.proto

```diff
@@ -28,10 +28,10 @@ option java_multiple_files = true;
 service HelloWorld {
 
   // Service Metadata - Name, version, id, RPC methods
-  option (uprotocol.name) = "example.hello_world";
-  option (uprotocol.version_major) = 1;
-  option (uprotocol.version_minor) = 0;
-  option (uprotocol.id) = 999;
+  option (uprotocol.service_name) = "example.hello_world";
```

Comment:
@halimragab -- These are the bits that I updated to get the hello_world_service.proto working with the 1.5.8 changes.

It seems like the other services need have these options updated too. Can you confirm?

Reply:
yes, I think  we just need to do find and replace all